cost-analyzer/templates/cost-analyzer-smtp-configmap.yaml

```
@@ -0,0 +1,23 @@
+{{- if .Values.kubecostProductConfigs }}
+{{- if .Values.kubecostProductConfigs.smtp }}
+{{- if .Values.kubecostProductConfigs.smtp.enabled }}
+# If the smtp.config is not specified, the configmap will not be created
+{{- if .Values.kubecostProductConfigs.smtp.config }}
+# If the secretname is specified, the configmap will not be created
+{{- if not .Values.kubecostProductConfigs.smtp.secretname }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ default "smtp-configs" .Values.smtpConfigmapName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{ include "cost-analyzer.commonLabels" . | nindent 4 }}
```

```suggestion
    {{- include "cost-analyzer.commonLabels" . | nindent 4 }}
```

When you include templates, you still need to chomp new lines at the beginning.